### PR TITLE
完善国际化调试代码

### PR DESCRIPTION
**Pull request by QKflame**
### <samp>🤖 Generated by Copilot at cb77462</samp>

This pull request adds custom i18n renderer components for input text and textarea fields, and a utility module to overwrite the default schema templates of AMIS components with the i18n components. The i18n components allow the user to translate the field values according to the locale, and to toggle between the corpus key and the plain text. The pull request also fixes the import paths of the custom renderer and utils modules in the Editor.tsx example file.

### <samp>🤖 Generated by Copilot at cb77462</samp>

> _`i18n` renderers_
> _added to AMIS schema_
> _autumn of languages_

### <samp>🤖 Generated by Copilot at cb77462</samp>

*  Added custom i18n renderer components for input text and textarea fields ([link](https://github.com/baidu/amis/pull/7814/files?diff=unified&w=0#diff-2785dd9e1974fd7b85e41c1dd9b6d4eabbe340328ad1de0556522edafec1cafcR1-R167), [link](https://github.com/baidu/amis/pull/7814/files?diff=unified&w=0#diff-78fdf5a6e23034d23262b4229bd933b386bc6552068ed9583fe83b4c61e9677fR1-R83))
*  Corrected the import paths of the custom renderer and utils modules in the `Editor.tsx` example file ([link](https://github.com/baidu/amis/pull/7814/files?diff=unified&w=0#diff-ac37ef7e74b39117226be066461e8c65edafc2318cf9874536ac80f09b6f9f41L12-R14))
*  Added a utility module to overwrite the default schema templates of various AMIS components with the custom i18n renderer components ([link](https://github.com/baidu/amis/pull/7814/files?diff=unified&w=0#diff-bc23893d1c6ff84ca839bafe12154b337dc664972eca2804f4c207de33357d22R1-R142))

### CODE REVIEW

File: packages/amis-editor/examples/renderer/InputTextI18n.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+ * @file client/renderer/InputTextI18n.tsx
+ * @description 支持国际化的 input-text 组件
+ */
+
+import {
+  FormControlProps,
+  FormItem,
+  makeTranslator,
+  autobind
+} from '../../../amis-core/src/index';
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/InputTextI18n.tsx:11]: 引用方式不对

File: packages/amis-editor/examples/renderer/InputTextI18n.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+import React from 'react';
+import {Icon, InputBox} from '../../../amis-ui/src/index';
+import {InputBoxProps} from '../../../amis-ui/src/components/InputBox';
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/InputTextI18n.tsx:14]: +2

File: packages/amis-editor/examples/renderer/InputTextI18n.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+  @autobind
+  onClickI18nBtn() {
+    const {value} = this.props;
+    if (corpusKeyReg.test(value!)) {
+      this.props?.onChange?.('');
+      this.props.onI18nChange?.('');
+      return;
+    }
+    let _uuid = 'i18n:1189fb5d-ac5b-4558-b363-068ce5decc99';
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/InputTextI18n.tsx:117]: uuid 是固定值？

File: packages/amis-editor/examples/renderer/InputTextI18n.tsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+      onChange,
+      value,
+      classPrefix,
+      className,
+      classnames,
+      placeholder,
+      disabled,
+      ...rest
+    } = this.props;
+    const {i18nEnabled} = getEnv((window as any).editorStore);
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/InputTextI18n.tsx:146]: 不可以依赖 window.editorStore

File: packages/amis-editor/examples/renderer/InputTextI18n.tsx [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+        placeholder={placeholder}
+        className={className}
+        disabled={disabled}
+      ></InputTextI18n>
+    );
+  }
+}
+
+@FormItem({
+  type: 'input-text-i18n'
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/InputTextI18n.tsx:165]: 编辑器相关的渲染器请遵循规范 `ae-`  打头

File: packages/amis-editor/examples/renderer/TextareaI18n.tsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+ * @description 支持国际化的 textarea 组件
+ */
+
+import React from 'react';
+import {
+  autobind,
+  FormControlProps,
+  FormItem
+} from '../../../amis-core/src/index';
+import {Icon, Textarea} from '../../../amis-ui/src/index';
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/TextareaI18n.tsx:12]: 引用方式

File: packages/amis-editor/examples/renderer/TextareaI18n.tsx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+            ></div>
+          ) : null}
+        </div>
+      </>
+    );
+  }
+}
+
+@FormItem({
+  type: 'textarea-i18n'
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/TextareaI18n.tsx:81]: +1

File: packages/amis-editor/examples/renderer/TextareaI18n.tsx [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+    return (
+      <>
+        <div
+          className={cx(
+            'TextareaI18n',
+            className,
+            'TextareaI18n-multi-container'
+          )}
+        >
+          <Textarea
```
Comment (2betop) [inline packages/amis-editor/examples/renderer/TextareaI18n.tsx:47]: 如果只有这不一样，有没有相关其他组合方式？

**Review by 2betop — CHANGES_REQUESTED** [review]

**Review by 2betop — APPROVED** [review]

**Review by igrowp — APPROVED** [review]

**Pull request merged**